### ISSUE TITLE
MELPA problem: company-go is never updated 

### ISSUE THREAD

**Issue by dennismayr** [melpa/melpa#5725]
Hi!

From a couple of weeks, I've observed that whenever I update ṕackages in Emacs, there's *always* an update for `company-go`, no matter if I've hit "update" 10 seconds after updating `company-go` for the last time. There is *always* an update for `company-go` no matter what, as if the last update from 10 seconds ago wasn't installed.

Not show-stopping, but rather curious nonetheless.

**Comment by milkypostman** [comment]
did you figure out whats going on?

**Comment by dennismayr** [comment]
Yes, I later realized it was caused by Spacemacs' develop branch: 

It duplicated the custom-config segment at the bottom of the dotfile, and added some options unavailable in the stable branch; when going back to the stable master branch, that seemed to confuse Emacs to the point of altering behavior and severely degrade performance.

I diffed the defaults dotfile vs mine, and weeded out duplicates and configurations for packages that weren't there anymore.

Problem disappeared, so I immediately closed this now non-issue.
